txpool/journal.go

```diff
@@ -0,0 +1,91 @@
+package txpool
+
+import (
+	"sync"
+
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+// TxStatus indicates the status of tx that is added to TxPool
+type TxStatus int8
+
+const (
+	TxSuccessful TxStatus = iota // Tx has been mined in a block successfully
+	TxDropped                    // Tx was dropped for some reason
+)
+
+var (
+	// txStatusToString is a mapping from TxStatus to text status
+	txStatusToString = map[TxStatus]string{
+		TxSuccessful: "success",
```

Comment:
I think the journal should be used only to signal when a transaction was dropped from the pool and why.

Reply:
it can be after we merge with 'eth_getTransactionReceipt'